xdrjson/conversion.go

```diff
@@ -24,6 +24,33 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Decode takes XDR binary, decodes it as the XdrType, returning XDR-JSON.
+//
+// If the XDR binary passed in is zero length, decoding will error.
+//
+// Returns the JSON message if decoding successful, otherwise an error.
+func Decode(xdrTypeName XdrType, xdrBinary []byte) (json.RawMessage, error) {
+	return convertAnyBytes(string(xdrTypeName), xdrBinary)
+}
+
+// DecodeInterface takes a value that serializes via a BinaryMarshaler
+// implementation to XDR binary, decodes the XDR binary and returns XDR-JSON.
+//
+// The XdrType that the XDR binary is decoded as is the XDR type name as
+// returned by the value's XdrJsonTypeName function.
+func DecodeInterface(xdr interface {
+	encoding.BinaryMarshaler
+	XdrJsonTypeName() string
```

Comment:
I assume this is a function will need to include in the bindings produced by xdrgen and that will require a PR for the xdrgen repo?

Reply:
Yup, that's correct.